.github/workflows/ci.yaml

```
@@ -182,6 +182,12 @@ jobs:
         with:
           arguments: integrationTestCosmos -Dscalardb.cosmos.uri=${{ secrets.COSMOS_URI }} -Dscalardb.cosmos.password=${{ secrets.COSMOS_PASSWORD }} -Dscalardb.cosmos.database_prefix=${{ env.db_prefix }}_
 
+      - name : Delete gradle daemon log files
+        if: always()
+#       Delete all files with the ".out.log" file extension in the "/home/azureuser/.gradle/daemon"
+#       folder hierarchy. These files accumulate over time and can end up using a lot of disk space
+        run : find /home/azureuser/.gradle/daemon -name "*.out.log" -type f -exec rm -vf {} +
```

I'm wondering if it's really okay to delete all the logs if we are running multiple integration tests at the same time. Maybe adding something like `-mtime +3` is safer?

```suggestion
        run : find /home/azureuser/.gradle/daemon -name "*.out.log" -type f -mtime +3 -exec rm -vf {} +
```

---

Indeed, thank you. I updated it.